cli/azd/pkg/auth/cache.go

```diff
@@ -36,6 +48,35 @@ func (a *msalCacheAdapter) Replace(ctx context.Context, cache cache.Unmarshaler,
 		return err
 	}
 
+	// In msal v1.0, keys were stored with mixed casing; in v1.1., it was changed to lower case.
+	// This handles upgrades where we have a v1.0 cache, and we need to convert it to v1.1,
+	// by normalizing the appropriate key entries.
+	c := map[string]json.RawMessage{}
+	if err = json.Unmarshal(val, &c); err == nil {
```

Comment:
Can you change this to the form of:
err = func()
if err != nil {
  log()
  return()
}

??  

That would save us one level of indentation here

Reply:
I've reduced the inner identation. I think it's somewhat important that this logic is clearly visible in the context of `Replace`. To adopt the proposed pattern, would need to introduce a new function.